src/CalculatedCharacteristics.Tests/CalculatedCharacteristics.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;net5.0</TargetFrameworks>

```

Comment:
netstandard soll drin bleiben wegen Kompatibilität. Wir können nicht voraussetzen, das alle schon .NET6 nutzen

Reply:
Ist in der Test-Assembly irrelevant.

Reply:
Ich hatte da immer Probleme mit. Aber wenn die Tests durchlaufen dann scheint's ja zu gehen.